Source/RapyutaSimulationPlugins/Private/Core/RRUObjectUtils.cpp

```diff
@@ -0,0 +1,144 @@
+#include "Core/RRUObjectUtils.h"
+
+// UE
+#include "Components/ActorComponent.h"
+#include "GameFramework/Actor.h"
+
+// RapyutaSimulationPlugins
+#include "Core/RRBaseActor.h"
+#include "Core/RRMathUtils.h"
+
+void URRUObjectUtils::SetupActorTick(AActor* InActor, bool bIsTickEnabled, float InTickInterval)
+{
+    // Tick if not required could be disabled to improve performance
```

Comment:
Just from curiosity, how much this can improve performance?
cc @chalice-graeme

Reply:
Ticks themselves are calls invoked frequently like ROS topic publication, thus generally, a just-enough policy should be encouraged in that sense I suppose.